src/layout/Navbar.tsx

```diff
@@ -5,99 +5,63 @@ import { Link } from 'react-router-dom';
-                                _hover={{ bg: '#8BA5FF' }}>
-                                <Box
-                                    justifyContent={'center'}
-                                    alignItems={'center'}
-                                    display={'flex'}
-                                    flexDirection={'column'}>
-                                    <FiHome size={30} title={'Hjem'} />
-                                    <Text>Hjem</Text>
-                                </Box>
-                            </Button>
-                        </Link>
-                    </Center>
-                </GridItem>
-                <GridItem h="100%" w="100%" alignSelf="center">
-                    <Link
-                        to="/statistikk"
-                        style={{
-                            textDecoration: 'none',
-                        }}>
-                        <Button
-                            h="100%"
-                            bg={'#6387ff'}
-                            borderRadius={70}
-                            w={'100%'}
-                            _hover={{
-                                bg: '#8BA5FF',
-                                border: '.1px solid #000',
-                                color: '#000',
-                            }}>
-                            Statistikk
-                        </Button>
-                    </Link>
-                </GridItem>
-            </Grid>
+            color="white"
+            marginTop={'10px'}
+            fontSize={'2rem'}>
+            <Link
+                style={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    height: '100%',
+                    width: '100%',
+                    background: '#8BA5FF',
+                    borderRadius: '3rem',
+                }}
+                to={'/energiflyt'}>
+                Energiflyt
+            </Link>
+            <Link
+                style={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    flexDirection: 'column',
+                    height: '135%',
+                    marginBottom: '25px',
+                    width: '30%',
+                    borderRadius: '3rem',
+                    border: '.1px solid black',
+                    background: '#8BA5FF',
+                }}
+                to={'/'}>
+                <FiHome size={30} />
+                <Text style={{ fontSize: '15px' }}>Hjem</Text>
+            </Link>
+            <Link
+                style={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    height: '100%',
+                    width: '100%',
+                    background: '#8BA5FF',
+                    borderRadius: '3rem',
```

Comment:
These `Link` elements have multiple shared attributes, would be cleaner if these where in a variable or an constum Link component with the shared styling.